CMakeLists.txt

```diff
@@ -22,15 +24,33 @@ set(tinyobjloader-examples-objsticher
 	${TINYOBJLOADEREXAMPLES_DIR}/obj_sticher/obj_sticher.cc
 	)
 
+#Install destinations
+set(TINYOBJLOADER_CMAKE_DIR lib/cmake)
```

Comment:
Can we please use either `${LIB_SUFFIX}` or GNUInstallDirs, so that if distros want to package this, they don't have to patch it?